### PR TITLE
Feature/jjdd8 hfv 3

### PR DESCRIPTION
**Pull request by dagiel**
Menu for terminal added with 2 functions already done.
All files in MenuConsola package.
Other changes:
App: Singeltone initialized and MainMenu.runMenu() added
HolidaysFilter: searchByName() and searchByDate methods changed into public

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshare/academy/highfive/App.java [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 
+    public static void singletonInitialization() throws IOException {
+        HolidaysSingleton.getInstance().initFromURL("https://calendarific.com/api/v2/holidays?&api_key=c2ddb57bb630fc01911bbcd01ae5907afaaced8e058cc0f33a938f517c0321e3&country=PL&year=2019");
+        HolidaysSingleton.getInstance().initSaveToFile("2019.json");
+    }
 
-    public static void main(String[] args) {
-        stdout.info("Start");
+    public static void main(String[] args) throws Exception {
+        singletonInitialization();
```
Comment (rafalmisiak) [inline src/main/java/com/infoshare/academy/highfive/App.java]: dajmy proszę formę czasownikową, pewnie initRepository() będzie wystarczające

File: src/main/java/com/infoshare/academy/highfive/MenuConsola/ConfigurationMenu.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    stdout.info("\nChange employees SORTING mode - UNDER CONSTRUCTION\n\n");
+                    break;
+                case 3:
+                    stdout.info("\nChange configuration of external file - UNDER CONSTRUCTION\n\n");
+                    break;
+                default:
+                    break;
+            }
+
+        } catch (InputMismatchException e) {
```
Comment (rafalmisiak) [inline src/main/java/com/infoshare/academy/highfive/MenuConsola/ConfigurationMenu.java]: To jest runtime exception którego nie łapiemy

File: src/main/java/com/infoshare/academy/highfive/MenuConsola/EmployeesManagingMenu.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                    stdout.info("\nAdd new employee - UNDER CONSTRUCTION\n\n");
+                    break;
+                case 3:
+                    stdout.info("\nRemove an employee - UNDER CONSTRUCTION\n\n");
+                    break;
+                default:
+                    break;
+            }
+
+        } catch (InputMismatchException e) {
```
Comment (rafalmisiak) [inline src/main/java/com/infoshare/academy/highfive/MenuConsola/EmployeesManagingMenu.java]: jw

File: src/main/java/com/infoshare/academy/highfive/MenuConsola/HolidaysPlanningMenu.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                    stdout.info("\nAdd vacation- UNDER CONSTRUCTION\n\n");
+                    break;
+                case 4:
+                    stdout.info("\nCancel vacation - UNDER CONSTRUCTION\n\n");
+                    break;
+                default:
+                    break;
+            }
+
+        } catch (InputMismatchException e) {
```
Comment (rafalmisiak) [inline src/main/java/com/infoshare/academy/highfive/MenuConsola/HolidaysPlanningMenu.java]: jw

File: src/main/java/com/infoshare/academy/highfive/MenuConsola/HolidaysReviewMenu.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                    stdout.info("\nDisplay employee vacation - UNDER CONSTRUCTION\n\n");
+                    break;
+                case 2:
+                    stdout.info("\nDisplay team vacations - UNDER CONSTRUCTION\n\n");
+                    break;
+                default:
+                    break;
+            }
+
+        } catch (InputMismatchException e) {
```
Comment (rafalmisiak) [inline src/main/java/com/infoshare/academy/highfive/MenuConsola/HolidaysReviewMenu.java]: jw

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshare/academy/highfive/MenuConsola/ConfigurationMenu.java (hunk 2 above)
Comment (dagiel) [reply src/main/java/com/infoshare/academy/highfive/MenuConsola/ConfigurationMenu.java]: Zrobione

File: src/main/java/com/infoshare/academy/highfive/MenuConsola/EmployeesManagingMenu.java (hunk 3 above)
Comment (dagiel) [reply src/main/java/com/infoshare/academy/highfive/MenuConsola/EmployeesManagingMenu.java]: zrobione

File: src/main/java/com/infoshare/academy/highfive/MenuConsola/HolidaysPlanningMenu.java (hunk 4 above)
Comment (dagiel) [reply src/main/java/com/infoshare/academy/highfive/MenuConsola/HolidaysPlanningMenu.java]: zrobione

File: src/main/java/com/infoshare/academy/highfive/MenuConsola/HolidaysReviewMenu.java (hunk 5 above)
Comment (dagiel) [reply src/main/java/com/infoshare/academy/highfive/MenuConsola/HolidaysReviewMenu.java]: zrobione

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**